lib/Server.js

```diff
@@ -1211,11 +1211,16 @@ class Server {
           res.write(`<h2>Compilation: ${name}</h2>`);
           res.write("<ul>");
 
-          const publicPath = item.publicPath === "auto" ? "" : item.publicPath;
+          const publicPath = item.publicPath === "auto" ? "/" : item.publicPath;
```

Comment:
"auto" means nothing before, can you provide example how reproduce the problem?

Reply:
- `cd ./examples/compression`
- `npx webpack serve --open`
- Go to `localhost:8080/webpack-dev-server`
- Click on main.js

Reply:
I think we can remove this line change, the code below will do the job.

This is more likely to be an issue with how `href` attribute works,

Reply:
Yep, maybe, I want to look at this in near future